returnn/torch/updater.py

```diff
@@ -102,11 +116,14 @@ def set_learning_rate(self, value):
         for param_group in self.optimizer.param_groups:
             param_group["lr"] = value
```

Comment:
Here you also need to assign `set_learning_rate`, such that you can use the correct value in `set_current_train_step`.